source/quark/R/01_QuarkData.R

```diff
@@ -78,7 +78,10 @@ QuarkData <- setRefClass("QuarkData",
                              nImps        = "integer",
                              compFormat   = "character",
                              miDatasets   = "ANY",
-                             miceObject   = "ANY"
+                             miceObject   = "ANY",
+			     sameNaCntVec = "vector",
```

Comment:
Do these three objects need to be fields in the QuarkData object? It looks like they are only used locally in the cleanCollinVars() function. Do they need to persist after cleanCollinVars() returns? If not please remove these fields and treat these variables as local.